evaluator/evaluate.go

```diff
@@ -57,6 +60,9 @@ func configure() {
 		DisableCompression: disableCompression,
 	}}
 	vmaasClient = vmaas.NewAPIClient(vmaasConfig)
+	enablePackageCache = utils.GetBoolEnvOrDefault("ENABLE_PACKAGE_CACHE", true)
```

Comment:
We will also need to add these vars to deploy/clowdapp.yaml file. Otherwise we won't be able to use them in openshift deployment.

Reply:
Added.